ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-3551-autoscaling.yaml

```diff
@@ -1,8 +1,8 @@
 base_images:
-  openshift_release_golang-1.17:
+  openshift_release_golang-1.18:
     name: release
     namespace: openshift
-    tag: golang-1.17
+    tag: golang-1.18
```

Comment:
This change will be reverted by the ci-operator. If you want to bump the version of go you need to update the openshift-ci/Dockerfile.tools in the integreatly-operator for this branch. Once that change is made these version will be updated.